src/encrypted_table/mod.rs

```diff
@@ -80,6 +78,14 @@ impl EncryptedTable<Headless> {
             .cts_config(&cts_config)
             .build_with_client_key()?;
 
+        Self::init_headless_with_zk_config(zerokms_config).await
+    }
+
+    pub async fn init_headless_with_zk_config(
```

Comment:
Minor suggestion here: mind spelling out `zerokms` here instead of abbreviating?
```suggestion
    pub async fn init_headless_with_zerokms_config(
```